### ISSUE TITLE
`/gazebo/default/pose/local/info` queue size

### ISSUE THREAD

**Issue by ivoysey** [schmerl/LLStaging#52]
we get this error pretty regularly and it seems bad.

```
Warning [Publisher.cc:134] Queue limit reached for topic /gazebo/default/pose/local/info, deleting message. This warning is printed only once.
```

**Comment by ivoysey** [comment]
i think this could actually be causing errors for us. i'm not sure how to change the size of a default topic's queue though.

**Comment by ivoysey** [comment]
talked to miguel about this. he's had a previous worry about it and spent some time with ivan attempting to track it down. they couldn't.

here's a post he made about it: http://answers.ros.org/question/243682/printing-data-from-a-subscriber-is-lagging-on-several-topics/

his general thought is that this is a systemic thing -- it's hard (impossible?) to change the queue size for built in queues, and this error is indicating a general overloading or lack of resources, rather than a particular problem with that one queue.

that is a problem but a hard one to solve or even know what to look for.

**Comment by schmerl** [comment]
It would be nice to know what amount of resources would make it go away. Then we could request it to LL. Let's put this on the "nice to have" list.

**Comment by ivoysey** [comment]
this doesn't go away if i bump to 10GB of ram locally. not sure about what ever configurations Lincoln has tried.

**Comment by Neels001** [comment]
Hi I am facing the same problem if u have found the solution, plz kindly suggest me..

**Comment by ivoysey** [comment]
We have yet to figure out a definitive solution. Sorry!

**Comment by Neels001** [comment]
Even if there is some workaround plz kindly suggest me...

**Comment by ivoysey** [comment]
Sorry, but no. There really isn't anything that I can offer you that I'm certain actually helps.

**Comment by Neels001** [comment]
Ok..Thank you :)

**Comment by orkunasa** [comment]
Is there any news or tips to deal with this problem?

**Comment by ivoysey** [comment]
@orkunasa Sorry, but no. We ended up ignoring this problem instead of really fixing it because it didn't effect the results that we were interested in.
